lib/simplifyExpression/basicsSearch/mixedNumber.js

```diff
@@ -0,0 +1,115 @@
+const Negative = require('../../Negative');
```

Comment:
I'd probably put it in `Node` since it's checking the node type and not really returning steps